lib/inline_oneboxer.rb

```diff
@@ -74,6 +73,12 @@ def self.lookup(url, opts = nil)
 
   private
 
+  def self.domain_is_blocked?(hostname)
+    SiteSetting.blocked_onebox_domains&.split('|').any? do |blocked|
```

Comment:
can this logic be reused by standard oneboxer as well?

Reply:
For sure. My focus was just on titles but I should explore outwards. Will start with the usages of ... `FinalDestination`